### PR TITLE
Update FAQ.md

### PR DESCRIPTION
**Pull request by celeneous**
Various grammar corrections and entry modifications
- Seleneous <3

### CODE REVIEW

**Comment by pyrosliced** [comment]
I like this because it doesn't overwrite anything I'm actively doing

File: FAQ.md [hunk 1]
```diff
@@ -60,41 +60,41 @@ Please restart the game.
...
+H refers to Host, which is a mod that is compatible with the original version. It only needs to be installed by the host, and other players can play without installing the mod. In order to be compatible with the original version, the H-series mods have many functions and roles that cannot be realized. Even if they are realized, the way of realization is very strange. For example, the sheriff is shown as an Impostor, and the instructions of gambling monsters cannot be hidden.
 
 ## The game ends inexplicably
 
-Game ends suddenly, If there are still imps alive and the victory conditions of the neutrals are not met. There are two possibilities:
+The game will end suddenly if there are still impostors alive and the victory conditions of the neutrals are not met. Two other may include possibilities:
```
Comment (0xDrMoe) [inline FAQ.md:75]: two other may include?

**Review by 0xDrMoe — CHANGES_REQUESTED** [review]
two other may include?

**Comment by pyrosliced** [comment]
I've since updated the FAQ as this was left quiet, close this perhaps?

**Pull request closed without merging**